elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1220,6 +1221,7 @@ public Resource toResource(final Supplier<Map<String, Relationship>> relationshi
                 : dictionary.getId(obj));
         resource.setRelationships(relationshipSupplier.get());
         resource.setAttributes(attributeSupplier.get());
+        resource.setLinks(getResourceLevelLink());
```

Comment:
@clayreimann @bduisenov, from the perspective of preserving current behavior I think we should probably make this configurable before finally merging this PR in. Playing off of the tri-state idea:

We could default the behavior to `PermitLinks` for the whole elide instance and unspecified beans. This way the behavior is the same for anyone pulling this change in.

I am assuming the following:

`ForbidLinks` -- Links is never displayed on a resource/elide instance
`PermitLinks` -- Links will be displayed if and only if requested by the caller (via a query param) for a resource/elide instance
`ForceLinks` -- Links are always displayed for a resource/elide instance

Provided those assumptions hold, I believe there shouldn't be much work in the configuration bit for this.

btw, thanks a lot for your work on this @bduisenov! :)